engine/common/tests/test_timezones.py

```diff
@@ -42,56 +41,3 @@ class MyCustomException(APIException):
 
     with pytest.raises(MyCustomException, match="Invalid timezone"):
         tz.raise_exception_if_not_valid_timezone("asdfasfd", exception_class=MyCustomException)
-
-
-class TestTimeZoneField:
-    @pytest.mark.parametrize("tz", pytz.all_timezones)
-    def test_valid_timezones(self, tz):
-        class MySerializer(serializers.Serializer):
-            tz = tz.TimeZoneField()
-
-        try:
-            serializer = MySerializer(data={"tz": tz})
-            serializer.is_valid(raise_exception=True)
-
-            assert serializer.validated_data["tz"] == tz
-        except Exception:
-            pytest.fail()
-
-    def test_invalid_timezone(self):
-        class MySerializer(serializers.Serializer):
-            tz = tz.TimeZoneField()
-
-        with pytest.raises(serializers.ValidationError, match="Invalid timezone"):
-            serializer = MySerializer(data={"tz": "potato"})
-            serializer.is_valid(raise_exception=True)
-
-    def test_it_works_with_allow_null(self):
-        class MySerializer(serializers.Serializer):
-            tz = tz.TimeZoneField(allow_null=True)
-
-        try:
-            serializer = MySerializer(data={"tz": None})
-            serializer.is_valid(raise_exception=True)
-            assert serializer.validated_data["tz"] is None
-
-            serializer = MySerializer(data={"tz": "UTC"})
-            serializer.is_valid(raise_exception=True)
-            assert serializer.validated_data["tz"] == "UTC"
-        except Exception:
-            pytest.fail()
-
-    def test_it_works_with_required(self):
-        class MySerializer(serializers.Serializer):
-            tz = tz.TimeZoneField(required=True)
-
-        with pytest.raises(serializers.ValidationError, match="This field is required"):
-            serializer = MySerializer(data={})
-            serializer.is_valid(raise_exception=True)
-
-        try:
-            serializer = MySerializer(data={"tz": "UTC"})
-            serializer.is_valid(raise_exception=True)
-            assert serializer.validated_data["tz"] == "UTC"
-        except Exception:
-            pytest.fail()
```

Comment:
relocated to a different test file, `engine/common/tests/test_custom_fields.py`, w/ other tests related to custom DRF serializer fields